02-peer-protocol.md

```diff
@@ -802,7 +825,12 @@ is destined, is described in [BOLT #4](04-onion-routing.md).
 A sending node:
   - MUST NOT offer `amount_msat` it cannot pay for in the
 remote commitment transaction at the current `feerate_per_kw` (see "Updating
-Fees") while maintaining its channel reserve.
+Fees") while maintaining its channel reserve
+  - if `option_anchor_outputs` applies to this commitment transaction and the sending
+    node is the funder:
+    - MUST be able to additionally pay for `to_local_anchor` and 
+      `to_remote_anchor` above its reserve. The anchor value should assumed to be
```

Comment:
This seems out of place and should be mentioned in BOLT 3 (size of the anchor outputs).

Reply:
It is already mentioned in BOLT 3 indeed. Removed.